### PR TITLE
Use `runBiDiServer.sh` as a single entry point

### PR DESCRIPTION
**Pull request by sadym-chromium**
The goal is to provide verbose logs of e2e run to help investigate failures if any.

### CODE REVIEW

File: src/bidiServer/index.ts [hunk 1]
```diff
@@ -130,12 +130,12 @@ async function onNewBidiConnectionOpen(
...
-    ? computeSystemExecutablePath({
-        browser: Browser.CHROME,
-        channel: chromeChannel,
-      })
-    : process.env['CHROME_BIN'];
+  const executablePath =
+    process.env['CHROME_BIN'] ??
+    computeSystemExecutablePath({
+      browser: Browser.CHROME,
+      channel: chromeChannel || ChromeReleaseChannel.DEV,
```
Comment (thiagowfx) [inline src/bidiServer/index.ts:137]:
```suggestion
      channel: chromeChannel ?? ChromeReleaseChannel.DEV,
```

Comment (thiagowfx) [reply src/bidiServer/index.ts:137]: Or, even better: Add a default argument to `chromeChannel` in the method signature.

**Review by thiagowfx — APPROVED** [review]

**Comment by sadym-chromium** [comment]
> The only thing that is missing -- but can be a separate PR, maybe I'll tackle it -- is to use runWPT.sh as a single entry point (as well), for the CI actions

Out of scope of this PR, as the goal was to provide e2e logging.

**Comment by thiagowfx** [comment]
I have another theory about the E2E error.

Let's try to downgrade `"@puppeteer/browsers": "0.1.1"` in package.json. It was changed ~4 days ago by myself, and it seems to coincide with the recent flakiness.

**Pull request merged**